designcompose/src/main/java/com/android/designcompose/squoosh/SquooshAnimate.kt

```diff
@@ -275,56 +279,31 @@ internal class SquooshAnimatedLayout(
 
 internal class SquooshAnimatedArc(
     target: SquooshResolvedNode,
-    private val from: Shape.Arc,
-    private val to: Shape.Arc,
+    private val from: ViewShape.VectorArc,
+    private val to: ViewShape.VectorArc,
     transition: AnimationTransition,
 ) : SquooshAnimatedItem(target, transition) {
     override fun apply(value: Float) {
         val iv = 1.0f - value
-        val arc =
-            Shape.Arc(
-                VectorArc(
-                    listOf(),
-                    listOf(),
-                    to.value.stroke_cap,
-                    from.value.start_angle_degrees * iv + to.value.start_angle_degrees * value,
-                    from.value.sweep_angle_degrees * iv + to.value.sweep_angle_degrees * value,
-                    from.value.inner_radius * iv + to.value.inner_radius * value,
-                    from.value.corner_radius * iv + to.value.corner_radius * value,
-                    to.value.is_mask,
-                )
-            )
-        // Unfortunately, the ViewData and View objects are also immutable.
+        val arc = vectorArc {
+            paths.clear()
```

Comment:
nit: I don't think we need to specifically clear the vectors. Their default value is already empty. I'd just cleared them in my other PR because I hadn't included the field names for the other fields.

Reply:
Done.